SUPPORT.md

```diff
@@ -0,0 +1,32 @@
+# Commercial Support for CloudNativePG
+
+CloudNativePG is an independent open-source project and does not officially
+endorse any specific company or service provider.
+
+However, to assist users in finding professional support, the
+"[Commercial Support](https://cloudnative-pg.io/support/)"
+page offers an alphabetical list of companies and individuals providing
+CloudNativePG-related products or services.
+
+*Please note that the CloudNativePG authors are not responsible for the accuracy
+or content provided by the listed companies or individuals.*
+
+## How to Get Listed
+
+To have your company or personal services featured on this list, please follow
+these steps to submit a pull request to the CloudNativePG website:
+
+1. **Organisation Name**: Clearly specify the name of your company or entity.
+2. **Organisation Logo**: Provide your company logo in SVG format.
+3. **Website Link**: Include a link to your homepage or a dedicated landing
+  page that explicitly mentions CloudNativePG support and includes at least one
+  link back to [cloudnative-pg.io](https://cloudnative-pg.io).
+4. **Organisation Description**: Submit a concise description of your company,
+  with a particular focus on your CloudNativePG and Kubernetes-related
+  professional support services (*maximum 400 characters*).
+
+[CloudNativePG maintainers will review each submission](https://github.com/cloudnative-pg/governance/blob/main/GOVERNANCE.md#voting)
+and reserve the right to vet your request if your website doesn’t clearly
+mention CloudNativePG support, includes at least one link back to
+[cloudnative-pg.io](https://cloudnative-pg.io), or if your claims cannot be
```

Comment:
Looking at this, it may be parsed (or at least I parsed it) as saying that requests
should NOT "include at least one link back".
IMO it's better to be redundant.

```suggestion
and reserve the right to vet your request if your website doesn’t clearly
mention CloudNativePG support, if it doesn't include at least one link back to
[cloudnative-pg.io](https://cloudnative-pg.io), or if your claims cannot be
```

Reply:
check now